features/step_definitions/civil_journey_steps.rb

```diff
@@ -500,3 +514,7 @@
   sleep 2
 end
 # rubocop:enable Lint/Debugger
+
+def create_bank_transactions
+  puts ">>>>>>>>>>>> CREATE ABAK DKRJALKSJDS #{__FILE__}:#{__LINE__} <<<<<<<<<<<<\n"
```

Comment:
Should this debug statement be replaced with actual method or should it be removed?